modules/ROOT/pages/to-manage-portal-resources.adoc

```diff
@@ -0,0 +1,49 @@
+= To Manage Portal Resources in Exchange
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Before starting
+
+To perform the following actions, you must login to the Anypoint Platform and obtain an authentication token. Once obtained, every following request must be made with the header `Authorization=bearer ACCESS_TOKEN`:
+
++
+[source,console]
+----
+ACCESS_TOKEN=$(curl https://anypoint.mulesoft.com/accounts/login -XPOST -d "username=YOUR_USERNAME&password=YOUR_PASSWORD" | jq -r ".access_token")
+----
++
+
+== Deleting Asset Portal images
+The number of resources that can be uploaded to an asset's portal is currently limited to 100 items. Attempting to add another image after reaching the limit will result in an error. In order to allow for more images to be created, you can consult the existing uploaded images and delete them by using the Exchange Experience API.
+
+To list all:
+    GET https://anypoint.mulesoft.com/exchange/api/v2/assets/:groupId/:assetId/:version/portal/draft/resources
+
+To see a specific image:
```

Comment:
To get a specific resource of an asset portal

Reply:
Fixed.